tests/test_subdomains.py

```diff
@@ -364,7 +367,10 @@ class Inner(SubDomainSet):
 
         assert((np.array(result) == np.array(fex.data[:])).all())
 
-    def test_multi_sets_eq(self):
+    @pytest.mark.parametrize('opt', ['advanced', 'blocking',
+                                     ('blocking', {'skewing': True}),
+                                     ('blocking', {'skewing': True, 'blockinner': True})])
+    def test_multi_sets_eq(self, opt):
```

Comment:
why not just two cases

```
['advanced',
 ('advanced', {'skewing': True})]
```

Reply:
One case out of four is redundant, though I need to be checking innermost skewing, in order not to harm subdomains, as subds appear in innermost position in other clusters.
Though I admit this issue was resolved since the introduction of SKEWABLE, cause of course subds are not TILABLE and therefore not SKEWABLE. If it is not an issue, I would prefer to keep the test with blockinner.

So from the four tests I can keep 3 but better not 2.